_posts/2020-09-26-bring-your-own-datatypes.md

```diff
@@ -1,8 +1,8 @@
 ---
```

Comment:
I can't seem to comment on the image file diff, but why did that file change? I assume you changed it from bfloat to posit?

Reply:
Nevermind, just opened the picture. Looks good!